_glossary/RNPM.md

```diff
@@ -0,0 +1,8 @@
+---
+title: rnpm (React Native Package Manager)
```

Comment:
Instead of having this addition everywhere, what do you think of adding `rnpm` to [the list of abbreviations](https://github.com/HugoGiraudel/SJSJ/blob/gh-pages/_data/abbr.yml) so we have proper helper display without cluttering the content with secondary information? 

If you agree, we can remove `(React Native Package Manager)` everywhere, including in the README.